vocode/streaming/action/external_actions_requester.py

```diff
@@ -52,6 +52,7 @@ async def send_request(
         payload: Dict[str, Any],
         signature_secret: str,
         additional_payload_values: Dict[str, Any] = {},
+        transport_options: Dict[str, Any] = {"retries": 2},
```

Comment:
could we actually just pass through the `AsyncHTTPTransport` object? is better for typing and configurability I'd say

Reply:
I considered it originally but thought it would introduce potentially undesirable coupling. But I think it's good to be opinionated about some things even if it introduces some coupling -- made the change.